keyboard.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>||Working with elements||</title>
+  <style>
+    #keyboard {
+      display: flex;
+    }
+    .white-key {
+      height: 100px;
+      width: 28px;
+      background-color: white;
+      outline-style: solid;
+      outline-width: thin
+    }
+
+    .black-key {
+      height: 60px;
+      background-color: black;
+      position: absolute;
+      outline-style: solid;
+      outline-width: thin
+    }
+  </style>
+</head>
+<body>
+  <div id="description"></div>
+  <div id="keyboard"></div>
+  <script> 
+    
+
+    document.body.onload = addWhiteKeys(14);
+    window.addEventListener('resize', resizeAllBlackKeys)
+    const allBlackKeysArray = Array.from(document.getElementsByClassName('black-key'))
+
+    function addWhiteKeys(numKeys) {
+      const whiteKeysPerOctave = 6
+      let i = 0;
+      let whiteKeyCount = 0;
+      while(i < numKeys) {
+        // create a new rectangle div element
+        const newDiv = document.createElement("div");
+        newDiv.setAttribute('id', `white-key${i}`)
+        newDiv.classList.add('white-key')
+    
+        // add the newly created element and its content into the DOM
+        const currentDiv = document.getElementById("description");
+      
+        document.getElementById('keyboard').appendChild(newDiv, currentDiv);
+
+        var coords = getPos(document.querySelector(`#white-key${i}`));
+
+        if(whiteKeyCount < whiteKeysPerOctave) {
+          if(blackKeyCheck(whiteKeyCount)) {
+            addBlackKey(`white-key${i}`, i, coords);
+          }
+          whiteKeyCount++
+        }
+        else whiteKeyCount = 0 
+          
+        i++
+      }
+
+      const totalKeyCount = document.getElementById("keyboard").childElementCount;
+      document.getElementById("description").innerHTML = `Below should be a keyboard with ${totalKeyCount} keys.`;
+    }
+
+    function addBlackKey(whiteKeyID, i, whiteKeyCoords) {
+      const newDiv = document.createElement("div");
+      newDiv.setAttribute('id', `black-key${i}`)
+      newDiv.classList.add('black-key')
+      document.getElementById('keyboard').appendChild(newDiv, whiteKeyID);
+      sizeBlackKey(document.getElementById(`black-key${i}`))
```

Comment:
you should be able to pass in `newDiv` here instead of querying for the element again
```suggestion
      sizeBlackKey(newDiv)
```